packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/__tests__/quanticGeneratedAnswer.test.js

```diff
@@ -48,6 +49,7 @@ function createTestComponent(options = defaultOptions) {
 const selectors = {
   initializationError: 'c-quantic-component-error',
   generatedAnswerCard: '[data-testid="generated-answer__card"]',
+  generatedAnswer: '.generated-answer__answer',
```

Comment:
Selector needs to be improved to be like the other selectors in this test, using the `data-testid` selector  is better than using a CSS class as a selector.